tests/vmi_configuration_test.go

```diff
@@ -2306,6 +2306,12 @@ var _ = Describe("Configurations", func() {
 			Expect(err).ToNot(HaveOccurred())
 			tests.WaitForSuccessfulVMIStart(vmi)
 
+			test_smbios := &cmdv1.SMBios{Family: "", Product: "", Manufacturer: ""}
+			smbiosJson, err := json.Marshal(test_smbios)
+			Expect(err).ToNot(HaveOccurred())
+			tests.UpdateClusterConfigValueAndWait(virtconfig.SmbiosConfigKey, string(smbiosJson))
+			defer tests.ClearKubeVirtConfigMap(virtconfig.SmbiosConfigKey)
```

Comment:
We can't blindly delete the SMBios settings. before we set them to something, we need to retrieve and store the original values. Once the test has concluded, we should restore that. It might make sense to do that in an AfterEach block.

It's possible that there wasn't anything set for SMBios, in which case Clearing the value is the correct response. However on clusters that did have this set before the test suite was run, this test would incur a side effect of deleting it.

Reply:
hmmm, is that needed for this test as well then? https://github.com/kubevirt/kubevirt/pull/3513/files#diff-d47bf036797965d742615e64adc3b777L2337-L2341

Reply:
I have updated this to use an `AfterEach` block to restore the ConfigMap SMBios settings to the originals at the beginning of the test